config/schedule.rb

```diff
@@ -96,7 +96,7 @@ def local(time)
   runner "RecordCounterJob.perform_async"
 end
 
-every :day, at: local("02:30 am"), roles: [:cron] do
+every :day, at: local("04:30 am"), roles: [:cron] do
```

Comment:
The matviews refresh in ~3 hours. We start at 12.30, which doesn't leave any margin.